verl/trainer/fsdp_sft_trainer.py

```diff
@@ -211,12 +211,16 @@ def _build_model_optimizer(self):
             use_meta_tensor=not config.tie_word_embeddings, mesh=self.device_mesh
         )
 
+        attn_implementation="flash_attention_2"
+        if self.device_name in ["mps", "cpu"]:
+            attn_implementation = "eager"
```

Comment:
I'd prefer that we add attn_implementation to https://github.com/volcengine/verl/blob/main/verl/trainer/config/config.py#L79 and let users configure this manually 

We can add some validation logic in `BaseModelConfig.__post_init__` that "eager" is expected for cpu & mps devices